teamd/teamd_runner_ttdp.c

```diff
@@ -1007,6 +1007,14 @@ static int elect_neighbor(struct teamd_context *ctx, struct ab *ab, uint8_t *nex
 			/* store winner mac */
 			memcpy(ab->elected_neighbor.neighbor_mac, ab->neighbors[candidate_suggestion].neighbor_mac,
 				sizeof(ab->elected_neighbor.neighbor_mac));
+			/* Update neighbor mac statefile */
+			/* Notify tcnd that something has changed */
```

Comment:
```suggestion
			/* Notify the rest of the stack that something has changed */
```

Probably should avoid using internal daemon names.